### PR TITLE
updating with circulars transition info

### PR DESCRIPTION
**Pull request by jracusin**
I updated slides at the end about the new circulars.  I largely took the text from the announcement.

Screenshot 2023-04-12 at 12 57 23 PM
Screenshot 2023-04-12 at 12 57 38 PM
Screenshot 2023-04-12 at 12 57 48 PM
Screenshot 2023-04-12 at 12 57 52 PM

### CODE REVIEW

**Review by lpsinger — APPROVED** [review]

**Comment by lpsinger** [comment]
This is great. We'll need more slides soon with walkthroughs of peer endorsements and so on, but this is a good start and is enough for tomorrow.

**Comment by jracusin** [comment]
> This is great. We'll need more slides soon with walkthroughs of peer endorsements and so on, but this is a good start and is enough for tomorrow.

Yeah, I agree that we'll need more for the webinar.  I was aiming for slides for tomorrow. @Vidushi-GitHub, you should make sure you're comfortable going through these.  You can probably skip over much of the intro material and Kafka demo, as we've presented that stuff before.  Focus on the "what's next" section.

**Comment by Vidushi-GitHub** [comment]
Is there any way to jump at slide 31 from slide 1? So that I can skip history, Kafka notices etc.

File: index.html [hunk 1]
```diff
@@ -767,24 +767,68 @@ <h1>Streaming GCN<br>Notices in Python</h1>
...
+            * Submit Circulars with our new Web form, or continue to submit by email.
           </textarea>
         </div>
       </div>
     </section>
-    
+
+    <section data-markdown>
+      <textarea data-template>
+        ## Here is what you need to know about preparing for the transition on April 17.
```
Comment (Vidushi-GitHub) [inline index.html:787]: remove 'here is'

April 17 is redundant!

File: index.html [hunk 2]
```diff
@@ -767,24 +767,68 @@ <h1>Streaming GCN<br>Notices in Python</h1>
...
+            <div>    
+              <img src="img/screen-shot-gcn-markdown.png" width="700" alt="Screen shot of GCN Circular web form">
+            </div>
+            <div data-markdown>
+              <textarea data-template>
+              ## Planned enhancements to GCN Circulars
+              * "Astro-Flavored Markdown"
+              * Automatically mint DOIs and BiBTeX entries
+              * Link your ORCID to your GCN Circulars
+              * Circulars will be available via Kafka topic
```
Comment (Vidushi-GitHub) [inline index.html:815]: My question: How Kafka topic will be helpful?

File: index.html [hunk 3]
```diff
@@ -767,24 +767,68 @@ <h1>Streaming GCN<br>Notices in Python</h1>
...
+      <section>
+          <div class="image-and-caption">
+            <div>    
+              <img src="img/screen-shot-gcn-markdown.png" width="700" alt="Screen shot of GCN Circular web form">
+            </div>
+            <div data-markdown>
+              <textarea data-template>
+              ## Planned enhancements to GCN Circulars
+              * "Astro-Flavored Markdown"
+              * Automatically mint DOIs and BiBTeX entries
```
Comment (Vidushi-GitHub) [inline index.html:813]: It isn't fully automatic and we haven't discussed anything with ADS later.

File: index.html (hunk 1 above)
Comment (jracusin) [reply index.html:787]: For tomorrow, it's worth reminding people over and over that the transition is on April 17.  For the webinars, we'll definitely want to remove it, as you suggest.

File: index.html (hunk 3 above)
Comment (jracusin) [reply index.html:813]: It's automatic for the old archive.  We need to follow-up with ADS to update this soon.  They are not minting DOIs, and none of it happens quickly.

**Pull request merged**